cookbooks/aws-parallelcluster-shared/spec/spec_helper.rb

```diff
@@ -76,4 +76,8 @@ def expect_to_include_recipe_from_resource(recipe, cookbook = 'any')
   expect_any_instance_of(Chef::RunContext).to receive(:include_recipe).with(recipe, { current_cookbook: cookbook })
 end
 
+def mock_get_vpc_cidr_list(cidr)
+  allow_any_instance_of(Object).to receive(:get_vpc_cidr_list).and_return(cidr)
+end
```

Comment:
Please avoid defining common functions when not strictly necessary: it might be confusing in the long run. In this case it's just one line of code, you can use it directly in your test instead of defining a function in the most shared code we have for ChefSpec tests.

Reply:
Removed the function as I no longer need to mock.